spa/src/components/dashboard/connectStripe/ConnectStripeToast.js

```diff
@@ -0,0 +1,66 @@
+import { useState, useCallback } from 'react';
+import * as S from './ConnectStripeToast.styled';
+
+import { withStyles } from '@material-ui/core/styles';
+import Tooltip from '@material-ui/core/Tooltip';
+
+// Assets
+import StripeLogo from 'assets/icons/stripeLogo.svg';
+import Triangle6Dots from 'assets/icons/triangle6Dots.svg';
+import RemoveIcon from '@material-ui/icons/Remove';
+
+const RETooltip = withStyles({
+  tooltip: {
+    color: '#fff',
+    backgroundColor: '#323232',
+    marginTop: '4px',
+    padding: '4px 6px',
+    fontSize: '10px',
+    borderRadius: '0.5px'
+  }
+})(Tooltip);
+
+const ConnectStripeToast = () => {
+  const [collapsed, setCollapsed] = useState(false);
+
+  const handleCollapse = useCallback(() => {
+    setCollapsed(true);
+  }, [setCollapsed]);
+
+  const handleExpand = useCallback(() => {
+    setCollapsed(false);
+  }, [setCollapsed]);
+
+  if (collapsed) {
+    return (
+      <RETooltip title="Connect to Stripe">
+        <S.ConnectStripeToastCollapsed data-testid={'connect-stripe-toast-collapsed'} onClick={handleExpand}>
```

Comment:
If it's just a string shouldn't it be `"` instead of `{'` ? I guess pre-comit doesn't change this, but in all the other component properties it's like that. Example: `<RETooltip title="Minimize" placement="bottom-end">`
```suggestion
        <S.ConnectStripeToastCollapsed data-testid="connect-stripe-toast-collapsed" onClick={handleExpand}>
```